rules.bzl

```diff
@@ -88,6 +88,8 @@ def _android_ndk_repository_impl(ctx):
             "{clang_resource_directory}": clang_resource_directory,
             "{sysroot_directory}": sysroot_directory,
             "{executable_extension}": executable_extension,
+            "{ndk_path}": ndk_path,
```

Comment:
Wouldn't this break remote caching if different developers install their NDKs in different locations?